### ISSUE TITLE
QuestPDF 2024.10.x Issues  

### ISSUE THREAD

**Issue by AndiRudi** [Relorer/HTMLToQPDF#25]
In the update process to target our application to .net 9, I have also upgraded the `QuestPDF` version from `2024.7.3` to `2024.10.4` and noticed a compilation issue. I have further tracked it down to start appearing with version `2024.10.3` so `2024.10.2` still works.

The reported error message is

```
The type 'IContainer' is defined in an assembly that is not referenced. You must add a reference to assembly 'QuestPDF, Version=2024.7.2.0, Culture=neutral, PublicKeyToken=null'.
```

This happens in this line
```
 column.Item().HTML(handler =>  handler.SetHtml(text));
```

I checked the release at https://github.com/QuestPDF/QuestPDF/compare/2024.10.3...main but could not find out what could cause this.

**Comment by DilushikaWeerawardhana** [comment]
I've encountered the same issue.
"QuestPDF" Version="2024.10.3"

**Comment by thepwrtank18** [comment]
Same issue here.

**Comment by vantageworks** [comment]
Still seems to be an issue on 2025.1.7? Just evaluating QuestPDf and HTMLtoQPDF, and get the same error

**Comment by yenn104** [comment]
Is there a way to solve this problem

**Comment by OlympicLarry** [comment]
> Is there a way to solve this problem

Yes, you can clone the repo and update the version it uses. Then, add a reference to the new HtmlToQPdf.dll in your project and remove this nugget package.

**Comment by kubalunacek** [comment]
Will be any update for this library to the current version of QuestPDF soon? Or I have to clone the repo and update it myself and using it as side project? Thanks
